### PR TITLE
Add consolidated release notes for 1.3.17

### PR DESCRIPTION
**Pull request by Divyaasm**
### Description
Add consolidated release notes for 1.3.17

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-release-notes-1.3.17.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-1.3.17.md).
+
+
+## MAINTENANCE
+
+
+### Opensearch Security
+
+
+* Update security reachout email ([#4333](https://github.com/opensearch-project/security/pull/4333))
```
Comment (peterzhuamazon) [inline release-notes/opensearch-release-notes-1.3.17.md:18]: I dont think this is related to the artifact, more like repo maintenance and changes.
Can remove.

File: release-notes/opensearch-release-notes-1.3.17.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+
+### Opensearch Security Dashboards Plugin
+
+
+* Update security reachout email ([#1948](https://github.com/opensearch-project/security-dashboards-plugin/pull/1948))
```
Comment (peterzhuamazon) [inline release-notes/opensearch-release-notes-1.3.17.md:24]: Same as above.

**Review by peterzhuamazon — APPROVED** [review]

File: release-notes/opensearch-release-notes-1.3.17.md (hunk 2 above)
Comment (Divyaasm) [reply release-notes/opensearch-release-notes-1.3.17.md:24]: So remove both security and security-dashboards ? @peterzhuamazon

Comment (peterzhuamazon) [reply release-notes/opensearch-release-notes-1.3.17.md:24]: Yes.

**Pull request merged**